packages/app/features/groups/GroupMetaScreen.tsx

```diff
@@ -33,21 +33,14 @@ export function GroupMetaScreen(props: Props) {
   });
   const canUpload = useCanUpload();
   const [showDeleteSheet, setShowDeleteSheet] = useState(false);
-  const { enabled, describe } = store.useLure({
-    flag: groupId,
-    inviteServiceEndpoint: INVITE_SERVICE_ENDPOINT,
-    inviteServiceIsDev: INVITE_SERVICE_IS_DEV,
-  });
 
   const handleSubmit = useCallback(
     (data: db.ClientMeta) => {
       setGroupMetadata(data);
       props.navigation.goBack();
-      if (enabled) {
-        describe();
-      }
+      store.createGroupInviteLink(groupId);
```

Comment:
why don't we need the `enabled` check here? How will this behave if it's a remote group that has them disabled?

Reply:
it will still work since it's the inviter who will send the invite, as long as they have permissions, which they should because we check permissions upon showing the button

Reply:
ohh, so we're counting on that "violate hosts grouper state" loophole we found earlier?